src/utils/getTranslatedTooltips.js

```diff
@@ -0,0 +1,235 @@
+const FI = {
+  draw: {
+    toolbar: {
+      actions: {
+        title: 'Peruuta piirros',
+        text: 'Peruuta'
+      },
+      finish: {
+        title: 'Piirros valmis',
+        text: 'Valmis'
+      },
+      undo: {
+        title: 'Poista edellinen piste',
+        text: 'Poista edellinen piste'
+      },
+      buttons: {
+        polyline: 'Piirrä vapaa viiva',
+        polygon: 'Piirrä monikulmio alue',
+        rectangle: 'Piirrä neliö',
+        circle: 'Piirrä ympyrä',
+        marker: 'Piirrä merkki',
+        circlemarker: 'Piirrä ympyrämerkki'
+      }
+    },
+    handlers: {
+      circle: {
+        tooltip: {
+          start: 'Klikkaa ja raahaa piirtääksesi ympyrä.'
+        },
+        radius: 'Radius'
+      },
+      circlemarker: {
+        tooltip: {
+          start: 'Klikkaa karttaa ja aseta ympyrämerkki.'
+        }
+      },
+      marker: {
+        tooltip: {
+          start: 'Klikkaa karttaa asettaaksesi merkki.'
+        }
+      },
+      polygon: {
+        tooltip: {
+          start: 'Klikkaa ja aloita muodon piirtäminen.',
+          cont: 'Klikkaa jatkaaksesi piirtämistä.',
+          end: 'Klikkaa ensimmäistä pistettä lopettaaksesi piirtäminen.'
+        }
+      },
+      polyline: {
+        error: '<strong>Virhe:</strong> muodon reunat eivät saa mennä ristiin!',
+        tooltip: {
+          start: 'Klikkaa aloittaaksesi viivan piirtäminen.',
+          cont: 'Klikkaa jatkaaksesi viivan piirtämistä.',
+          end: 'Klikkaa viimeistä pistettä päättääksesi piirtäminen.'
+        }
+      },
+      rectangle: {
+        tooltip: {
+          start: 'Klikkaa ja raahaa piirtääksesi neliö.'
+        }
+      },
+      simpleshape: {
+        tooltip: {
+          end: 'Vapauta painike lopettaaksesi piirtäminen.'
+        }
+      }
+    }
+  },
+  edit: {
```

Comment:
Is it possible to also translate `Zoom in` and `Zoom out` somewhere in here?

Reply:
Unfortunately this is limitation caused by react-leaflet and thus impossible :(

Reply:
1. Can't recreate this, did you run yarn again after changing to this branch?

2. Darn, I mistook this for another bug. This is impossible to accomplish at the moment, because subsection views are actually QuestionView and the editor lives in HearingView. This would be achieved if QuestionView and HearingView were to be refactored into a single view (as it should be), but this is huge refactoring and shouldn't be part of this PR.

Reply:
Yeah agreed, refactoring is a separate issue, I take it we must defer it to https://github.com/City-of-Helsinki/kerrokantasi-ui/issues/487

And yes, probably yarn was the issue here, because trying again, I cannot reproduce the error anymore :)